TestBuildRefineryPatrolVars_FullConfig fails on main after #3194 merge

## Description

`TestBuildRefineryPatrolVars_FullConfig` fails on current `main` after PR #3194 was merged. The test expects 5 vars from `DefaultMergeQueueConfig()` but `buildRefineryPatrolVars` now emits 7, because #3194 added `judgment_enabled` and `review_depth` to the output without updating the test expectations.

## Failure

```
=== RUN   TestBuildRefineryPatrolVars_FullConfig
    patrol_helpers_test.go:166: expected 5 vars, got 7: [target_branch=main
    integration_branch_refinery_enabled=true integration_branch_auto_land=false
    run_tests=true delete_merged_branches=true judgment_enabled=false review_depth=standard]
--- FAIL: TestBuildRefineryPatrolVars_FullConfig (0.00s)
```

## Impact

Every PR rebased onto current `main` will hit this CI failure. It's unrelated to the PR's own changes, which makes it confusing for contributors.

## Root cause

Commit 8fea7b84 (PR #3194) added `judgment_enabled` and `review_depth` to `buildRefineryPatrolVars` in `prime_molecule.go` and added the config fields/accessors in `config/types.go`, but did not update `patrol_helpers_test.go` to expect the two new vars.

## Fix

Update the `expected` map in `TestBuildRefineryPatrolVars_FullConfig` to include `"judgment_enabled": "false"` and `"review_depth": "standard"`.

--- a/internal/cmd/patrol_helpers_test.go
+++ b/internal/cmd/patrol_helpers_test.go
@@ -126,7 +126,8 @@ func TestBuildRefineryPatrolVars_FullConfig(t *testing.T) {
 	vars := buildRefineryPatrolVars(ctx)
 
 	// DefaultMergeQueueConfig: refinery_enabled=true, auto_land=false, run_tests=true,
-	// test_command="" (language-agnostic), target_branch="main" (from rig config), delete_merged_branches=true
+	// test_command="" (language-agnostic), target_branch="main" (from rig config),
+	// delete_merged_branches=true, judgment_enabled=false, review_depth="standard"
 	// New commands (setup, typecheck, lint, build) default to empty = omitted
 	expected := map[string]string{
 		"integration_branch_refinery_enabled": "true",
@@ -134,6 +135,8 @@ func TestBuildRefineryPatrolVars_FullConfig(t *testing.T) {
 		"run_tests":                           "true",
 		"target_branch":                       "main",
 		"delete_merged_branches":              "true",
+		"judgment_enabled":                    "false",
+		"review_depth":                        "standard",
 	}
 
 	varMap := make(map[string]string)

